nip47/publish_nip47_info.go

```diff
@@ -56,6 +56,14 @@ func (svc *nip47Service) PublishNip47Info(ctx context.Context, relay nostrmodels
 		}
 		capabilities = svc.permissionsService.GetPermittedMethods(&app, lnClient)
 		permitsNotifications = svc.permissionsService.PermitsNotifications(&app)
+
+		// NWA: associate the info event with the app so that the app can receive the wallet pubkey
+		tags = append(tags, []string{"p", app.AppPubkey})
+		// also include the LN address for apps that use it (e.g. nostr apps assign lud16 to the profile)
+		lightningAddress, _ := svc.albyOAuthSvc.GetLightningAddress()
```

Comment:
Remove this, leaks metadata and allows searching by relay for multiple connections by a lightning address